app/helpers/users_helper.rb

```diff
@@ -11,4 +11,9 @@ def first_and_last_name_initials(user)
   def full_name(user)
     "#{user.first_name + " " + user.last_name}"
   end
+  
+  def all_users_in_organisation(user)
```

Comment:
@apeksha2697  please think of proper method name, here you are saying all_users_in_organisation, but below you are checking on a single user

Reply:
done